### ISSUE TITLE
Upgrade DHIS2 header bar

### ISSUE THREAD

**Issue by jakub-bao** [pepfar-datim/datim-approvals#139]
I had some REACT warnings related to the dhis2 header bar.
Viktor explained to me they don't support the lib any more and we should upgrade to `ui-widgets`:

https://github.com/dhis2/ui-widgets

![image](https://user-images.githubusercontent.com/47790868/70813486-87924a00-1d97-11ea-84a4-83b8797f9dfa.png)

cc @tomzemp

**Comment by jakub-bao** [comment]
Plenty of examples here: https://jira.dhis2.org/browse/DHIS2-7617

**Comment by jakub-bao** [comment]
Good examples from Viktor:

`index.js` https://github.com/dhis2/user-app/blob/master/src/index.js
`App.js` https://github.com/dhis2/user-app/blob/master/src/App.js

**Comment by jakub-bao** [comment]
@benguaraldi So I upgraded the `HeaderBar`.
The old one is retired and not maintained.

The only problem is that the new one has visual look & feel of DHIS2 2.33 but the rest of our system is on 2.30.

Can we leave it this way or should we postpone this upgrade after DHIS2 2.33 upgrade?

![image](https://user-images.githubusercontent.com/47790868/70820259-39387780-1da6-11ea-91c9-905deab27b91.png)

**Comment by benguaraldi** [comment]
Interesting.  I'd vote for keeping the old header bar for a little bit, but maybe @hardingt or @hkim224 has a different opinion?

**Comment by jakub-bao** [comment]
Actually, it's more complicated than that...

This is what we got.

![image](https://user-images.githubusercontent.com/47790868/70820797-8406bf00-1da7-11ea-8156-a559e7062520.png)

1. Native DHIS2 2.30 HeaderBar
2. Custom app HeaderBar I am able to do using the old legacy libraries
3. New HeaderBar I can do using the new updated libraries

I personally like **3.**  more then I like **2.**.

Also, let me try to update to DATIM logo and check if I can enable to search bar.
Then the new HeaderBar would be very close to the native 2.30.

**Comment by benguaraldi** [comment]
I guess I'm not sure why we don't just live with the react warnings for a few months and then upgrade the header bar when it's upgrading for all of the other apps?

**Comment by jakub-bao** [comment]
@benguaraldi Sure, I'll freeze this branch and can merge it in later.
I was just afraid of merge conflicts later on but should be OK.

FYI, we'll have to manually upgrade all apps and their header bars if we want to be consistent after `2.33` upgrade.

BTW> Is 2.33 upgrade still pending around Feb 2020?

**Comment by jakub-bao** [comment]
Freezing this now until `2.33` upgrade. 
I've put back the old header bar.

Note to myself:
Easily switch to `2.33` by replacing:
![image](https://user-images.githubusercontent.com/47790868/70826985-42314500-1db6-11ea-95c5-983fa212cf88.png)

And then removing `@dhis2/d2-ui-header-bar` from devDependencies:
![image](https://user-images.githubusercontent.com/47790868/70827031-5f661380-1db6-11ea-8fef-7c2c57cbec3f.png)

These libs have to stay:
```
    "@dhis2/ui-core": "^4.1.1",
    "@dhis2/ui-widgets": "^2.0.5",
    "@dhis2/app-runtime": "^2.0.4",
    "@dhis2/d2-i18n": "^1.0.6",
    "d2": "^31.8.0",
```

**Comment by jakub-bao** [comment]
Done. It will be tested and released as a part of #142
